src/routes/(site)/blog/[id]/+page.svelte

```diff
@@ -5,9 +5,29 @@
   import { Temporal } from '@js-temporal/polyfill';
   import Labels from '$lib/components/blog/labels.svelte';
   import BlogBody from './blog-body.svelte';
+  import { MetaTags } from 'svelte-meta-tags';
+
   export let data: PageData;
+  let description: string;
+  if (data.post.bodyText) {
+    description = data.post.bodyText.substring(0, 100);
+  }
 </script>
 
+<MetaTags
+  openGraph={{
+    title: data.post.title,
+    description: description,
```

Comment:
not seem to work, can you look into why?

![image](https://user-images.githubusercontent.com/8981287/200957751-77e98b5c-fb36-4f78-a3b6-572f9df5a946.png)